### PR TITLE
aarch64 related changes

### PR DESCRIPTION
**Pull request by qookei**

### CODE REVIEW

File: bootstrap.d/media-libs.yml [hunk 1]
```diff
@@ -489,7 +490,7 @@ packages:
         - '--buildtype=debugoptimized'
         #- '--buildtype=debug'
         - '-Dglx=dri'
-        - '-Dplatforms=wayland,x11'
+        - '-Dplatforms=wayland,x11,drm'
```
Comment (Dennisbonke) [inline bootstrap.d/media-libs.yml:493]: We should bump the revision for this package.

File: bootstrap.d/sys-libs.yml [hunk 2]
```diff
@@ -77,6 +81,7 @@ packages:
         - '--with-pkg-config-libdir=/usr/lib/pkgconfig'
         - '--with-termlib'
         - '--enable-widec'
+        - '--disable-stripping'
```
Comment (Dennisbonke) [inline bootstrap.d/sys-libs.yml:84]: We should bump the revision for this package.

File: bootstrap.yml [hunk 3]
```diff
@@ -826,7 +877,10 @@ packages:
         - '-DLLVM_LINK_LLVM_DYLIB=ON'
         # RTTI affects the ABI. Hence, we enable it.
         - '-DLLVM_ENABLE_RTTI=ON'
-        - '-DLLVM_TARGETS_TO_BUILD=X86'
+        - '-DLLVM_TARGETS_TO_BUILD=X86;AArch64'
+        - '-DLLVM_TARGET_ARCH=@OPTION:arch@'
+        - '-DLLVM_DEFAULT_TARGET_TRIPLE=@OPTION:arch-triple@'
+        - '-DLLVM_HOST_TRIPLE=@OPTION:arch-triple@'
```
Comment (Dennisbonke) [inline bootstrap.yml:883]: We should bump the revision for this package.

File: bootstrap.d/dev-lang.yml [hunk 4]
```diff
@@ -113,11 +113,12 @@ packages:
       - xz-utils
       - gdbm
       - util-linux
+      - libintl
```
Comment (Dennisbonke) [inline bootstrap.d/dev-lang.yml:116]: We should bump the revision for this package.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Unsure about the change back to libressl as discussed on Discord

**Review by Dennisbonke — APPROVED** [review]

**Comment by Dennisbonke** [comment]
LGTM now.

File: bootstrap.d/media-libs.yml [hunk 5]
```diff
@@ -276,8 +276,7 @@ packages:
         - '-DCMAKE_INSTALL_DOCDIR=/usr/share/doc/libjpeg-turbo-2.0.5'
         - '-DCMAKE_INSTALL_DEFAULT_LIBDIR=lib'
         - '-DWITH_JPEG8=ON'
-        # HACK!! Hardcoding this makes it non-portable
-        - '-DCMAKE_SYSTEM_PROCESSOR=amd64'
+        - '-DCMAKE_SYSTEM_PROCESSOR=@OPTION:arch@'
```
Comment (avdgrinten) [inline bootstrap.d/media-libs.yml:279]: Is arm64 equivalent to x86_64 for this purpose? (The new option is set to x86_64.)

Comment (qookei) [reply bootstrap.d/media-libs.yml:279]: Yes, libjpeg-turbo's CMake file accepts both x86_64 and amd64

**Comment by avdgrinten** [comment]
Alright, then let's merge this.

**Pull request merged**